tests/containers/podman_netavark.pm

```diff
@@ -0,0 +1,147 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman, netavark, aardvark
+# Summary: Test podman netavark network backend
+# Maintainer: qac team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use version_utils qw(package_version_cmp is_transactional);
+use containers::utils qw(get_podman_version registry_url);
+use transactional qw(trup_call check_reboot_changes);
+use utils qw(zypper_call);
+use Utils::Systemd qw(systemctl);
+
+sub remove_subtest_setup {
+    assert_script_run("podman container rm -af");
+    assert_script_run("podman network prune -f");
+    assert_script_run("podman network ls");
+    assert_script_run("podman ps -a");
```

Comment:
is this just a debug log?

Reply:
Well, I have found it useful to double check whether the clean up ran fine in case of further issues with the test case

Reply:
Do you mean checking it manually in stdout? because this command, if everything is cleaned up or not, would still result in a zero exit code, right?

Reply:
Yes, while working on the code, I have found it useful for a test reviewer to print the current setup networks as I am recycling certain parameters (IPs,MACs etc.) between the test objects. This part certainly not necessary from the test perspective, but might save some time during the test review in case something goes wrong.

Reply:
Got it, I was missing that. Thanks for clearing it up

Reply:
Would it make sense to do a `validate_script_output` to make sure we are not getting any output?

Reply:
Isn't it debug output?

Reply:
ok, made changes to validate the output of the clean-up.
http://kepler.suse.cz/tests/20933#step/podman_netavark/92